tests/unit/test_main.py

```diff
@@ -70,5 +73,44 @@ def test_loadstore(self):
         store = pv.store(dataset_id = self.id, filename = self.h5, count = 10)
         self.assertTrue(isinstance(store, VedaBase))
 
-    def test_createfromgeojson(self):
-        pass
+    @patch('pyveda.main.from_geo')
+    def test_createfromgeojson(self, from_geo):
+        ''' Test the create_from_geojson method '''
+        # TODO: create fixtures for this kind of thing?
+        mock_doc = {
+            "name":              'mockname',
+            "classes":          [],
+            "dataset_id":       '234',
+            "dtype":            np.dtype('uint8'),
+            "userId":           'foo',
+            "imshape":          [],
+            "releases":         'releases',
+            "mltype":           'classification',
+            "tilesize":         [],
+            "image_refs":       [],
+            "sensors":          [],
+            "bounds":           [],
+            "public":           True,
+            "count":            3,
+            "percent_cached":   1,
+            "background_ratio": 1.0
+        }
+        from_geo.return_value = {'properties': mock_doc}
+        geojson = {}
+
+        class _Rda():
+            def graph(self):
+                return {'nodes': [{'id': 'someid'}]}
+
+        class _Image():
+
+            def __init__(self):
+                self.dtype = np.dtype('uint8')
+                self.shape = [256, 256, 3]
+                self.rda = _Rda()
+                self.rda_id = 'someotherid'
+
+        image = _Image()
+
+        vcp = pv.create_from_geojson(geojson, image, mock_doc['name'])
+        self.assertTrue(isinstance(vcp, VedaCollectionProxy))
```

Comment:
python unit has all the checks builtin: use `assertIsInstance(...)` for these kind of checks; failures will report more accurately

Reply:
cool I can correct that